includes/abstracts/abstract.llms.admin.metabox.php

```diff
@@ -303,6 +304,9 @@ public function output_errors() {
 		}
 
 		foreach ( $errors as $error ) {
+			if ( is_object( $error ) && 'WP_Error' === get_class( $error ) ) {
```

Comment:
```suggestion
			if ( is_wp_error( $error ) ) {
```

Prefer WP Core methods when available